web/src/app/util/SetFavoriteButton.tsx

```diff
@@ -54,8 +54,8 @@ export function SetFavoriteButton({
       <IconButton
         aria-label={
           isFavorite
-            ? `Unset as a Favorite ${typeName}`
-            : `Set as a Favorite ${typeName}`
+            ? `Unset as a Favorite ${typeName.toLowerCase()}`
+            : `Set as a Favorite ${typeName.toLowerCase()}`
```

Comment:
Maybe we can use `_.startCase` so it says `Set as Favorite Escalation Policy` instead of `Set as Favorite escalationpolicy`

```suggestion
            ? `Unset as a Favorite ${_.startCase(typeName)}`
            : `Set as a Favorite ${_.startCase(typeName)}`
```

Reply:
addressed, applied .toLowerCase() to account for Service, Rotation, Schedule checks